src/pages/endpoint/index.tsx

```diff
@@ -132,12 +132,11 @@ const Endpoint = () => {
 
         // handle website status changes
         const new_website_status = { ...website_status, clients_country: values.clients_country }
-        isBrowser() && localStorage.setItem('manual_clients_country', values.clients_country)
```

Comment:
No longer using these cookies? We still remove it on L124. May want to remove there as well.